release/sdk-config.yaml.tmpl

```
@@ -43,6 +43,10 @@ commands:
   path: daml-helper/daml-helper
   desc: "Launch the Sandbox"
   args: ["run-jar", "--logback-config=daml-sdk/sandbox-logback.xml", "daml-sdk/daml-sdk.jar", "sandbox"]
+- name: sandbox-next
+  path: daml-helper/daml-helper
+  desc: "Launch the Sandbox's upcoming implementation (for testing purposes)"
```

```suggestion
  desc: "Launch the Sandbox's upcoming implementation (experimental, for testing purposes)"
```
We use that for other features (e.g. triggers) so given that this also seems fairly experimental at this stage it seems helpful to make that explicit.